sigs/rules-authors/CHARTER.md

```diff
@@ -0,0 +1,71 @@
+# SIG Rules Authors
+
+## Objective
+
+Authors of commonly used Bazel rulesets can share technical approaches for solving common problems, speak with a coherent voice when interacting with the core Bazel team, and provide a more consistent experience for Bazel end-users.
+
+> The term "rulesets" is used to refer to all Bazel extensions, including Editor Plugins and Starlark shared libraries.
+
+## Participating
+
+### As a contributor
+
+Membership has been originally granted to maintainers of the "canonical" rulesets: those in the bazelbuild GitHub org, or listed in the [recommended rulesets](https://docs.bazel.build/versions/4.1.0/rules.html#recommended-rules).
+
+Criteria for new members is still under discussion: <https://github.com/bazel-contrib/SIG-rules-authors/issues/1>
+
+### As a company
+
+The SIG is funded by companies that rely on the community-maintained rulesets.
+We need your support to continue providing the software you depend on!
+
+> As of November 2021, Google is _not_ a funder of the SIG.
```

Comment:
Given that you point out Google here, what are your criteria for calling out companies funders of the SIG in future?

We will participate to the group meetings and our involvement so far has been and will be investing time.